### PR TITLE
import_csv method takes File object instead of file path

### PR DESCRIPTION
**Pull request by radimsuckr**
`import_csv` method now takes File objects instead of file paths.

### CODE REVIEW

File: chamber/importers/__init__.py [hunk 1]
```diff
@@ -13,12 +13,11 @@
     from itertools import izip_longest as zip_longest
 
 
-def simple_count(filename, encoding):
+def simple_count(file):
     # Encoding must be passed to match encoding used in the importer.
```
Comment (Formulka) [inline chamber/importers/__init__.py]: you should probably remove this, the argument is not there anymore

File: chamber/importers/__init__.py [hunk 2]
```diff
@@ -43,19 +42,18 @@ class AbstractCSVImporter:
     delimiter = ';'
     encoding = 'utf-8'
 
-    def __call__(self, csv_path):
+    def __call__(self, file):
         """csv_path is a required parameter as calling the function without CSV does not make sense"""
```
Comment (Formulka) [inline chamber/importers/__init__.py]: this should be changed as well to respect the new arguments

**Comment by radimsuckr** [comment]
> Doesn't this break some functionality since you've changed the importer? chamber might be used by other projects. Either way update all docstrings.

Yeah, this change is backward incompatible. The importers now take file-like object instead of file path. But luckily this change is not that big of a deal.

File: chamber/importers/__init__.py [hunk 3]
```diff
@@ -43,19 +40,28 @@ class AbstractCSVImporter:
...
-                reader,
-                row_count=simple_count(self.get_filename(custom_csv_path=custom_csv_path), encoding=self.get_encoding())
-            )
+    def __call__(self, file):
+        """file is a required parameter as calling the function without CSV file does not make sense"""
+        self.import_csv(file)
+
+    def import_csv(self, file=None):
+        if not file:
+            with open(self.csv_path, encoding=self.get_encoding()) as file:
```
Comment (radimsuckr) [inline chamber/importers/__init__.py]: @matllubos Is loading from disk really a needed feature or should we just drop it entirely and keep only the file-like object way.

**Pull request merged**